ocaml/idl/datamodel.ml

```diff
@@ -170,7 +170,7 @@ let _pvs_proxy = "PVS_proxy"
 let _pvs_cache_storage = "PVS_cache_storage"
 let _feature = "Feature"
 let _sdn_controller = "SDN_controller"
-
+let _vdi_nbd_server_info = "vdi_nbd_server_info"
```

Comment:
This is a really minor point, but we could consider capitalizing the acronyms in the name of this class, like we do for the class names above: `VDI_NBD_server_info`. What do you think?

Reply:
I wondered about it. I'm not sure. But I don't think we ever see these strings anywhere else anyway.

Reply:
I think we see them in the auto-generated docs and API bindings, as the class name? But in this case it's not important at all I guess, because this function doesn't have any messages/constructors. But it would probably be more consistent to capitalize it, as most, or maybe all of the others are capitalized?